feat: harvester-ui-extension requires upgrade for harvester v1.4.1 release (#711)
harvester/docs: docs/upgrade/v1-4-0-to-v1-4-1.md

For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).

> ### 1. Upgrade Harvester UI Extension to v1.0.3 on Rancher v2.10.1
> 
> To import Harvester v1.4.1 on Rancher v2.10.1, harvester-ui-extension requires upgrade to v1.0.3.
> 
> Follow below steps to upgrade harvester-ui-extension

Reviewer: ```suggestion
### Update Harvester UI Extension on Rancher v2.10.1

To import Harvester v1.4.1 clusters on Rancher v2.10.1, you must use **v1.0.3** of the Rancher UI extension for Harvester.
```

> 1. Go to Rancher UI `local -> Apps -> Repositories` page, find harvester repository with URL **https://github.com/harvester/harvester-ui-extension** and branch **gh-pages** then click `refresh` action
> 
> ![](/img/v1.4/upgrade/rancher-2.10.1-repository-page.png)
> 

Reviewer: ```suggestion
1. On the Rancher UI, go to **local > Apps > Repositories**.

1. Locate the repository named **harvester**, and then select **⋮ > Refresh**.

    This repository has the following properties:
    
    - URL **https://github.com/harvester/harvester-ui-extension**
    - Branch: **gh-pages**

    ![](/img/v1.4/upgrade/rancher-2.10.1-repository-page.png)
```

> 2. Go to extension page, click `update` and choose version `1.0.3`
> 
> ![](/img/v1.4/upgrade/update-harvester-ui-extension-modal.png)
> 

Reviewer: ```suggestion
1. Go to the **Extensions** screen.

1. Locate the extension named **Harvester**, and then click **Update**.

1. Select version **1.0.3**, and then click **Update**.

    ![](/img/v1.4/upgrade/update-harvester-ui-extension-modal.png)
```

> It will display error message after installing new version. The error message will auto dismiss after waiting a few moment and refresh the page. 
> 
> This is a known issue on Rancher v2.10.0 and v2.10.1 and will be fixed in v2.10.2. 

Reviewer: ```suggestion
1. Allow some time for the extension to be updated, and then refresh the screen.

:::info important

The Rancher UI displays an error message after the extension is updated. The error message disappears when you refresh the screen.

This issue, which exists in Rancher v2.10.0 and v2.10.1, will be fixed in v2.10.2. 

:::
```

Related issues:
  - [[BUG] v1.4.0 -> v1.4.1-rc1 upgrade stuck in Pre-drained and the node stay in Cordoned](https://github.com/harvester/harvester/issues/7366)
  - [[IMPROVEMENT] Cleanup orphaned volume runtime resources if the resources already deleted](https://github.com/longhorn/longhorn/issues/6764)

### 2. Upgrade Harvester UI Extension to 1.0.3 on Rancher v2.10.1

> To import Harvester v1.4.1 on Rancher v2.10.1, harvester-ui-extension requires upgrade to version `1.0.3`.

Reviewer 2: Do we need to clarify that Harvester v1.0.3 only works with Rancher v2.10.1 or higher?
